plexpy/datafactory.py

```diff
@@ -888,6 +890,39 @@ def calc_most_concurrent(title, result):
                                    'stat_title': 'Most Concurrent Streams',
                                    'rows': most_concurrent})
 
+            elif stat == 'top_libraries':
+                top_libraries = []
+
+                try:
+                    query = 'SELECT section_id, section_name, section_type ' \
+                        'FROM library_sections'
+
+                    result = monitor_db.select(query)
+                except Exception as e:
+                    logger.warn("Tautulli DataFactory :: Unable to execute database query for get_home_stats: top_libraries: %s." % e)
+                    return None
+
+                library_data = libraries.Libraries()
+
+                for item in result:
+                    library_item = library_data.get_watch_time_stats(section_id=item['section_id'], grouping=None , query_days=time_range)
+
+                    row = {
+                        'total_plays': library_item[0]['total_plays'],
+                        'total_duration': library_item[0]['total_time'],
+                        'type': item['section_type'],
+                        'library_name': item['section_name']
+                    }
```

Comment:
To be consistent with the rest of the API, this should use `section_type` and `section_name` instead of `type` and `library_name`.

The following values should also be included.

```
section_id
last_play
thumb
art
```

See `get_library_stats` below for the logic to get the thumb and art.

I don't remember exactly why I added all the additional blank values (see top users), but I think it was for some API compatibility reason.

Reply:
Good point with the naming convention.
I saw those empty values at different `stat`s, but sure I will add them.